pkg/api/v1alpha1/clustercurator_types.go

```diff
@@ -0,0 +1,85 @@
+// +kubebuilder:object:generate=true
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	runtime "k8s.io/apimachinery/pkg/runtime"
+)
+
+// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
+// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+
+// ClusterCuratorSpec defines the desired state of ClusterCurator
+type ClusterCuratorSpec struct {
+	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
+	// Important: Run "make" to regenerate code after modifying this file
+
+	// This is the desired curation flow
+	DesiredCuration string `json:"desiredCuration,omitempty"`
+
+	// Points to the Cloud Provider or Ansible Provider secret
+	ProviderCredentialPath string `json:"providerCredentialPath,omitempty"`
```

Comment:
`path`? I think hive used a lot of `Ref` for secret names. 
And is it expecting the `<namespace>/<name>` format? may be we should also mention in the comment

Reply:
I'm not concerned about this one as I'm going to remove it as we are handling the credential copy in the Console